### ISSUE TITLE
About the training time

### ISSUE THREAD

**Issue by yifliu3** [zeng-yifei/STAG4D#2]
Dear devs,

Thanks for your great work which has inspired me a lot. But I am confused about the training time for 1 hour as mentioned in the paper. Does this time refers to total time for training all samples or training one sample. If training one sample, which procedure takes so long?

Looking forward to your reply!

**Comment by zeng-yifei** [comment]
What do you mean by 'training all samples' and 'training one sample'? Do you mean training all the 4D results for different cases and training one 4D result for only one case?

Actually, one hour corresponds to the time of training one 4D content. Though rendering a picture is fast in 3DGS, it still takes some time to produce a SDS loss in the backpropagation of diffusion model.

**Comment by yifliu3** [comment]
Yes, I was wondering about the time of training one 4D result. Thanks a lot for your reply.
